staging/src/k8s.io/api/apps/v1/types.go

```diff
@@ -272,6 +272,7 @@ type StatefulSetStatus struct {
 
 	// Total number of available pods (ready for at least minReadySeconds) targeted by this statefulset.
 	// This is a beta field and enabled/disabled by StatefulSetMinReadySeconds feature gate.
+	// +optional
```

Comment:
And `omitempty` should be added into the json tag. So do the beta apis.

Reply:
no, we only want to change the openapi schema requiredness, not any serialization